src/wampy.js

```diff
@@ -1350,144 +1350,115 @@ class Wampy {
 
     /**
      * Handles websocket invocation message event
-     * @param {object} data - decoded event data
+     * @param {Array} data - decoded event data array
      * @private
      */
-    async _onInvocationMessage (data) {
-        if (!this._rpcRegs[data[2]]) {
-            // WAMP SPEC: [ERROR, INVOCATION, INVOCATION.Request|id, Details|dict, Error|uri]
-            this._send([WAMP_MSG_SPEC.ERROR, WAMP_MSG_SPEC.INVOCATION,
-                data[1], {}, 'wamp.error.no_such_procedure']);
-            return this._log(WAMP_ERROR_MSG.NON_EXIST_RPC_INVOCATION);
-        }
-
+    async _onInvocationMessage ([, requestId, topic, details, argsList, argsDict]) {
```

Comment:
@lucasrmendonca Another one nitpick: when you agree with a comment or suggestion and fix it, mark the conversation as `resolved` (see the button below `Resolve conversation`). In this case it will be easier for me, as reviewer to navigate through the PR and see that all comments are resolved and we can merge.